kolibri/plugins/base.py

```diff
@@ -10,10 +10,10 @@
 logger = logging.getLogger(__name__)
 
 
-class MandatoryPluginMethodNotImplemented(Exception):
+class MandatoryPluginMethodNotImplemented(NotImplementedError):
 
     def __init__(self):
-        NotImplemented.__init__(self, "Plugin needs to define this method")
+        super(MandatoryPluginMethodNotImplemented, self).__init__("Plugin needs to define this method")
```

Comment:
Have you tested this part? It's the part I couldn't get working, and I think I did the above...